CreateSigningDirectory.cmd

```diff
@@ -0,0 +1,55 @@
+::Build NuGet packages step
+@ECHO off
+SETLOCAL
+
+set BinaryOutputDirectory=%1
+set Configuration=%2
+set Platform=%3
+
+if "%BinaryOutputDirectory%" EQU "" (
+set BinaryOutputDirectory=%~dp0bld\bin
+)
+
+if "%Configuration%" EQU "" (
+set Configuration=Release
+)
+
+if "%Platform%" EQU "" (
+set Platform=AnyCPU
+)
+
+set BinaryOutputDirectory=%BinaryOutputDirectory%\%Platform%_%Configuration%\
+set SigningDirectory=%BinaryOutputDirectory%\..\Signing
```

Comment:
>set [](start = 0, length = 3)

If you set `SigningDirectory` first, you won't have to back up a level :-)

Reply:
The same in the other batch file.

---
In reply to: [209795461](https://github.com/Microsoft/jschema/pull/21#discussion_r209795461) [](ancestors = 209795461)

Reply:
right will fix this up.

---
In reply to: [209796248](https://github.com/Microsoft/jschema/pull/21#discussion_r209796248) [](ancestors = 209796248,209795461)